src/v3/src/components/InfoBox/InfoBox.tsx

```diff
@@ -53,15 +53,16 @@ const InfoBox: UISchemaElementComponent<{
         // eslint-disable-next-line react/jsx-props-no-spreading
         {...({ 'data-se': dataSe })}
         className={`infobox-${messageClass.toLowerCase()}`}
+        role="alert"
```

Comment:
also do we always want the role to be alert?  Even if its not an ERROR class message like INFO?

Reply:
> also do we always want the role to be alert? Even if its not an ERROR class message like INFO?

For these types of messages I believe so because it will prompt the screen reader to prioritize reading the content in that info box. 

Not sure why the role was added here, I am going test to make sure it still shows up, if so I will remove it

Reply:
I confirmed this has the role as expected so I removed this.